route/build/src/standalone/get_basin_runoff.f90

```diff
@@ -319,4 +342,60 @@ SUBROUTINE timeMap_sim_forc(tmap_sim_forc,     & ! inout: time-steps mapping dat
 
  END SUBROUTINE timeMap_sim_forc
 
+
+ ! *********************************************************************
+ ! private subroutine: scale and offset of forcing input
+ ! *********************************************************************
+ SUBROUTINE scale_forcing (forcing_data_in,      & ! inout: array of forcing, runoff, evaporation or precipitation
+                           a,                    & ! in: scale factor
+                           b)                      ! in: offset values
+
+  USE public_var,     ONLY: verySmall          ! directory containing input data
+  USE public_var,     ONLY: realMissing        ! real missing value
+  USE dataTypes,      ONLY: inputData          ! input data class keeps runoff, evaporation, and precipitation
+
+
+  implicit none
+  ! input, output
+  class(inputData), intent(inout)   :: forcing_data_in    ! forcing data structure
+  REAL(dp), INTENT(in)              :: a, b
```

Comment:
Better names for the variables than a,b. And also define them with a comment.

Reply:
Should be scale and offset based on the calls.

Reply:
done